view-samples/imdb-card-view/README.md

```diff
@@ -0,0 +1,47 @@
+# Flashcards
+
+## Summary
+
+This example transforms list items into cards formatted with IMDB Movie listing layout. To ensure the view functions correctly, make sure all specified columns are included. The icons are referenced from Office UI Fabric library (https://uifabricicons.azurewebsites.net/)
+
+The Director and Actor names are hyperlinks so that you can leverage them to filter other movies where they might have featured. You can achieve that by using query strings as described in [this blog](https://sudeepghatak.com/using-hyperlinks-in-list-view-json-to-apply-filters/)
+
+![screenshot of the sample](./assets/screenshot.jpg)
+
+## View requirements
+
+Column Name                 | Type
+----------------------------|-----------------------------------------
+Title                       | Single line of text
+Year                        | Single line of text
+Duration                    | Single line of text
+Rating                      | Choice
```

Comment:
I was wondering what options to set when trying this sample. If you have any suggestions or examples of choices, I would appreciate it if you could add them.

Reply:
This comment is a comment on the choice column.

Reply:
Added. Thanks